temp_functions.py

```diff
@@ -0,0 +1,75 @@
+import read_db
+import pandas as pd
+import time
+from tqdm import tqdm
+
+class functions():
+
+    def __init__(self):
+        self.df=None
+        self.dict={}
+
+    def set_dataframe(self):
+        extract = read_db.ExtractData()
+        self.df = extract.get_timeslots_info()
+
+    def main(self):
+        self.set_dataframe()
+        temp = {}
+        for i,row in tqdm(self.df.iterrows()):
+            key = row["EmployeeID"], row["RelationID"]
+
+            ## Calculate the remaining availability in each calendar month and week
+            appointmentLength = (row["UntilUtc"] - row["FromUtc"]).total_seconds()/3600
+            if key[0] in temp:
+                if temp[key[0]]["DateLastWorked"].year == row["FromUtc"].year:
+                    if temp[key[0]]["DateLastWorked"].month == row["FromUtc"].month:
```

Comment:
Dit is niet helemaal goed omdat je ook een maand van 25ste tot 25ste kan hebben

Reply:
> Dit is niet helemaal goed omdat je ook een maand van 25ste tot 25ste kan hebben

Het is kalendermaand afgewerkt, dit is een makkelijke fix en dit wilde ik de eerstvolgende keer dat we afspreken even opbrengen.

Reply:
Zou het echt op contractdatum doen, dat is het netste!